src/student_success_tool/analysis/pdp/dataops.py

```diff
@@ -18,6 +18,7 @@ def make_student_term_dataset(
     peak_covid_terms: set[tuple[str, str]] = constants.DEFAULT_PEAK_COVID_TERMS,
     num_terms_checkin: t.Optional[int] = None,
     key_course_subject_areas: t.Optional[list[int]] = None,
+    key_courses: t.Optional[list[int]] = None,
```

Comment:
I _almost_ added this feature myself, back when I added `key_course_subject_areas`. Thanks for taking it on!